server/socket.js

```diff
@@ -63,40 +65,56 @@ export default function startSocketServer(http) {
     });
 
 
-    socket.on(CS.ADD_MESSAGE, data => {
-      Message.add(data, (err, result) => {
-        io.to(data.channelId).emit(SC.ADD_MESSAGE, result.toObject());
+    socket.on(CS.ADD_MESSAGE, ({senderId, channelId, text}) => {
+      if (!(
+        validate(senderId).isString().inRange(20, 30).end() &&
+        validate(channelId).isString().inRange(20, 30).end() &&
+        validate(text).isString().inRange(0, 1000).end()
```

Comment:
This limit should match the character limit of the message box. I think it's a good idea to extend it to 1000 char.